tests/saw/spec/handshake/s2n_handshake_io.cry

```diff
@@ -221,6 +221,7 @@ type handshake = {handshake_type : [32]
                  ,message_number : [32]
                  }
 
+type HANDSHAKE_ACTION_TOTAL = 21
```

Comment:
Maybe STATE_MACHINE_SIZE? OR TOTAL_HANDSHAKE_ACTIONS? TOTAL_MESSAGE_TYPES?

It might also make more sense if you declare it down where all the handshake types are declared, since it'll always be one more than the highest message type id.

Reply:
Good idea! Check the new commit.